src/pages/workspace/workflows/ToggleSettingsOptionRow.tsx

```diff
@@ -0,0 +1,85 @@
+import React, {useState} from 'react';
+import {View} from 'react-native';
+import type {ViewStyle} from 'react-native';
+import useStyleUtils from '@hooks/useStyleUtils';
+import type {SvgProps} from 'react-native-svg';
+import Icon from '@components/Icon';
+import Switch from '@components/Switch';
+import Text from '@components/Text';
+import useThemeStyles from '@hooks/useThemeStyles';
+import useWindowDimensions from '@hooks/useWindowDimensions';
+import ImageSVG from '@components/ImageSVG';
+import Dot from '@assets/images/dot.svg';
+
+type ToggleSettingOptionRowProps = {
+    icon: React.FC<SvgProps>;
+    title: string;
+    subtitle: string;
+    hasBeenToggled: boolean;
+    onToggle: (isEnabled: boolean) => void;
+    subMenuItems?: React.ReactNode;
+    isEndOptionRow?: boolean;
+};
+
+function ToggleSettingOptionRow({icon, title, subtitle, onToggle, subMenuItems, isEndOptionRow, hasBeenToggled}: ToggleSettingOptionRowProps) {
+    const [isEnabled, setIsEnabled] = useState(hasBeenToggled);
+    const styles = useThemeStyles();
+    const StyleUtils = useStyleUtils();
+
+    const toggleSwitch = () => {
+        setIsEnabled(!isEnabled);
+        onToggle(!isEnabled);
+    };
+    const {isSmallScreenWidth} = useWindowDimensions();
+
+    function VerticalDots({count}: {count: number}) {
+        return (
+            <View style={StyleUtils.getWorkspaceWorkflowsDotStyle(isEnabled, isSmallScreenWidth) as ViewStyle}>
+                {Array.from({length: count}, () => (
+                    <ImageSVG
+                        contentFit="contain"
+                        src={Dot}
+                        height={isSmallScreenWidth ? 4 : 6}
+                        width={isSmallScreenWidth ? 4 : 6}
+                        pointerEvents="none"
+                    />
+                ))}
+            </View>
+        );
+    }
+
+    return (
+        <View style={styles.pRelative}>
+            <View style={StyleUtils.getWorkflowsStyle('container')}>
+                <View style={StyleUtils.getWorkflowsStyle('content')}>
+                    <Icon
+                        src={icon}
+                        height={48}
+                        width={48}
+                        additionalStyles={StyleUtils.getWorkflowsStyle('icon')}
+                    />
+                    <View style={StyleUtils.getWorkflowsStyle('timelineOverride')} />
+                    {!isEndOptionRow && (
+                        <VerticalDots count={7}/>
+                    )}
```

Comment:
This is not a requirement in the DD, I don't think it's a problem. 

Otherwise, it would be _complicated_ and _time consuming_ to achieve this. 
We already have issues because of React Native limitations (no background size, no radial gradient available...).

cc @mountiny @luacmartins

Reply:
I would not say its not requirement in the design doc. The design doc cannot capture all nuances of the design, but we want to make sure the UI looks good in all various window sizes.

However, I agree this is complicated and its not worth you digging into how to make this work the best. I think we should remove the Dots from the scope of this PR and create external issue with mocks and let contributors handle this cc @trjExpensify @shawnborton

Reply:
I think so yeah, I don't think they add a huge amount to the experience to warrant holding this PR on them. I'd also be open to just leaving them completely, but I'll defer to the design team on that.

Reply:
Resolving this comment after Shawn answered [here](https://github.com/Expensify/App/pull/36345#issuecomment-1956588423)

TLDR: we're removing dots